app/utils/store.py

```diff
@@ -74,7 +74,8 @@ def get_document_metadata(self, service_id, document_id, decryption_key):
             )
 
             return {
-                'mimetype': metadata['ContentType']
+                'mimetype': metadata['ContentType'],
+                'verify_email': True if metadata.get('Metadata', {}).get('hashed-recipient-email', '') else False
```

Comment:
in the last get, is there a reason for alternative being an empty string instead of `None`, or does it just not matter much?

Reply:
Definitely could be `None` - and maybe it should 😅 This way it's only one type - always a string - but probably None is clearer so I'll update it :)

(However functionally it doesn't matter)